app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -359,4 +393,21 @@ public void onMicrolocationsDownloadDone(MicrolocationDownloadEvent event) {
 
     }
 
+    @Subscribe
+    public void ErrorDialog(ErrorDialogEvent event){
+        downloadProgress.setVisibility(View.GONE);
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle("Error")
+                .setMessage(event.s1 + ": " + event.s2)
+                .setNeutralButton("OK", new DialogInterface.OnClickListener() {
```

Comment:
You should pass `null` as the listener if nothing is being done, again strings go into a separate file.